lib/zstd_stats.h

```diff
@@ -0,0 +1,164 @@
+/*
+    zstd - standard compression library
```

Comment:
"debug" is ignored in .gitignore and can give problems with Visual Studio (debug/release)